Bottom: afa3c5ff4fac5a3cf145c86914ed679fd2176565
Top:    fc6c66b06a4d291001181a5e3866d2b91c274c7f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-20 23:31:46 +0100

Refresh of hid-common-draw-the-hole-outli

---

diff --git a/src/crosshair.c b/src/crosshair.c
index e724537..2c1e68c 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -361,7 +361,7 @@ XORDrawBuffer (BufferTypePtr Buffer)
     PinType moved_via = *via;
     moved_via.X += x; moved_via.Y += y;
 
-    gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_via, false, false);
+    gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_via, true, false);
   }
   END_LOOP;
 }
